app/controllers/user_rules_controller.rb

```diff
@@ -0,0 +1,72 @@
+class UserRulesController < ApplicationController
+  def index
+    authorize workflow
+
+    rules = UserRule.where(workflow_id: params[:workflow_id])
```

Comment:
workflow.user_rules?

Reply:
i think i actually made that change in a commit that's in a subsequent PR, good catch